packages/mapviewer/src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -161,6 +161,13 @@ export function createLayerObject(parsedLayer, currentLayer, store, featuresRequ
         )
     }
 
+    // Attention the uuid that is automatically generated in the AbstractLayer class is not always sufficient
+    // if we create multiple times the same layer (e.g. with different timestamps) the already created uuid would be the same for both layers
+    // that's why whe have to create a new uuid here
```

Comment:
Are you referring to the use of the duplicate button here? If so I think this could be solved by editing a bit the `clone` function as said in my previous comment (and maybe make this addition of code irrelevant)